src/utils/localStorageUtils.js

```diff
@@ -0,0 +1,39 @@
+import GiantSwarm from 'giantswarm';
+
+export const removeUser = () => localStorage.removeItem('user');
+
+export const fetchSelectedOrganizationFromStorage = () => {
+  return localStorage.getItem('app.selectedOrganization');
+};
+
+export const fetchUserFromStorage = () => {
+  let user;
+
+  try {
+    user = JSON.parse(localStorage.getItem('user'));
+  } catch (e) {
+    user = {
+      auth: {},
+    };
+  }
+
+  // User was logged in pre-jwt auth being available.
+  // Migrate.
+  if (user && user.authToken) {
+    user.auth = {
+      scheme: 'giantswarm',
+      token: user.authToken,
+    };
+  }
+
+  return user;
+};
+
+export const setUser = userData => {
```

Comment:
I'm worried that maybe we dont' always have `userData.auth.token` and `userData.auth.scheme`, so we might 'unset' the authentication details of the api client... need to think a bit about this

Reply:
Also I think this could maybe be better described as `saveUser` or `persistUser`

especially if we also add the organization setting method here, then it is nice to have some kind of verb that implies persistence, since that's what this file is about.

Reply:
IMO using __set__ and __localStorage__ toghether is fine, I mean I think it's clear what it does, but we can change it if you think it's clearer!

Reply:
Yeah the naming isn't a big deal, but I just kinda wish in the other files, it would be clearer. So in this file it's ok, but where it's being implemented, because of the way we import it, it's hard to know if this is some local function or something imported, and then what package it's coming from. 

But I'm ok with the name for now to be honest, not a big deal.

Reply:
Aah, ok I didn't understand you... you're right... we can rename it when importing it also

Reply:
Changed method names and added `To/FromStorage`suffix to all of them, following what we already have in the `fetch...` methods